aeon/dj_pipeline/docs/notebooks/Example_queries_exp0.1.ipynb

```diff
@@ -1,224 +0,0 @@
-{
```

Comment:
@ttngu207 @iamamutt hey guys, is there a reason these example notebooks were deleted?